pygmt/tests/baseline/test_plot3d_matrix_color.png.dvc

```diff
@@ -1,4 +1,4 @@
 outs:
-- md5: 9b1e94da314923794dca11d20ae3e5df
+- md5: d08eaf4bf196c8680a3b7c0d8f4259a7
   size: 24071
```

Comment:
`test_plot3d_matrix_color` still failing on Windows. See https://github.com/GenericMappingTools/pygmt/pull/1322/checks?check_run_id=2756778343#step:11:486:

```
__________________________ test_plot3d_matrix_color ___________________________
Error: Image dimensions did not match.
  Expected shape: (1040, 1729)
    D:\a\pygmt\pygmt\tmp-test-dir-with-unique-name\results\pygmt.tests.test_plot3d.test_plot3d_matrix_color\baseline.png
  Actual shape: (991, 1729)
    D:\a\pygmt\pygmt\tmp-test-dir-with-unique-name\results\pygmt.tests.test_plot3d.test_plot3d_matrix_color\result.png
```

| baseline (correct) | result (wrong) |
|--|--|
| ![baseline](https://user-images.githubusercontent.com/23487320/120921477-fb58ad00-c717-11eb-993d-2c64a95ae82e.png) | ![result](https://user-images.githubusercontent.com/23487320/120921480-fdbb0700-c717-11eb-9721-f5efa2a4826b.png) |

Reply:
But it passes on Linux and macOS. I think we can mark it as xfail on Windows, and see if @joa-quim can help debug the Windows-only issue.

Reply:
I've marked `test_plot3d_matrix_color` as xfail on Windows (and reverted the dvc baseline image change to that test) at 327013474302b2c08fe4cb0756d70f91d221920c.